console/network/src/lib.rs

```diff
@@ -160,6 +160,11 @@ pub trait Network:
     /// The maximum number of outputs per transition.
     const MAX_OUTPUTS: usize = 16;
 
+    /// The maximum program depth.
+    const MAX_PROGRAM_DEPTH: usize = 1024;
+    /// The maximum number of imports.
+    const MAX_IMPORTS: usize = 2048;
```

Comment:
Can you provide some context on why these depths and imports are where they're at?

Based on our design, I believe we're constraining other parameters already at the 32 mark. It would make sense to constraint both of these parameters alongside the 32 mark.

Reply:
The `MAX_PROGRAM_DEPTH` was decided based off of Ethereum's MAX_CALL_DEPTH.
I went with this as a healthy upper bound. I see no issue with restricting `MAX_PROGRAM_DEPTH` to 32.

`MAX_IMPORTS` should be larger because a user may want to create a program that reads the mappings of a large number of other programs.

Reply:
Ah actually, suppose that `program1` reads `program2` reads ... reads `program32`
If we `program0` to reads `program1`, then we need a greater program depth.

Reply:
Discussed offline, waiting for a change to 64 and 64